internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -263,7 +246,7 @@ func (rp *RegistryPod) podForBundleRegistry() (*corev1.Pod, error) {
 // getContainerCmd uses templating to construct the container command
 // and throws error if unable to parse and execute the container command
 func (rp *RegistryPod) getContainerCmd() (string, error) {
-	const containerCommand = "/bin/mkdir -p {{ .DBPath }} &&" +
+	const containerCommand = "/bin/mkdir -p {{ .DBPath | dirname }} &&" +
```

Comment:
NB: this may cause a similar issue in OCP described [here](https://github.com/operator-framework/operator-sdk/pull/3856#issuecomment-688855166).